themes/openy_themes/openy_carnation/src/scss/presentation/_variables.scss

```diff
@@ -0,0 +1,116 @@
+//
+// Colours
+// --------------------------------------------------
+
+// grays
+$white:    #fff !default;
+$gray-100: #ebebeb;
+$gray-200: #ccc;
+$gray-300: #979797;
+$gray-400: #636466;
+$gray-500: #4f4f4f;
+$gray-600: #333333;
+$gray-700: #2f2f2f;
+$gray-800: #777777;
+//$gray-900: #212529 !default;
+$black:    #000 !default;
+
+// openy colour palette
+$sky-blue:            #bfe1f3;
+$light-blue:          #00aeef;
+$blue:                #0089d0;
+$dark-blue:           #0060af;
+$pink:                #c6168d;
+$indigo:              #92278f;
+$purple:              #5c2e91;
+$red:                 #ed1c24;
+$orange:              #f15922;
+$yellow:              #fcaf17;
+$green:               #20bdbe;
+$teal:                #006b6b;
+$cyan:                #01a490;
+
+$bright-green:        #69ff00;
+$green-aa:            #008485;
+$dark-green-aa:       #007677;
+
+$colors: () !default;
+$colors: map-merge((
+    "sky-blue":       $sky-blue,
+    "light-blue":     $light-blue,
+    "dark-blue":      $dark-blue,
+), $colors);
+
+// theme colours
+$primary:           $blue !default;
+$primary-darker:    $dark-blue;
+
+$theme-colors: () !default;
+$theme-colors: map-merge((
+    "pink":               $pink,
+    "indigo":             $indigo,
+    "purple":             $purple,
+    "primary-darker":     $primary-darker,
+    "gray-100":           $gray-100,
+    "gray-200":           $gray-200,
+    "gray-300":           $gray-300,
+    "gray-400":           $gray-400,
+    "gray-500":           $gray-500,
+    "gray-600":           $gray-600,
+    "black":              $black,
+), $theme-colors);
+
+// Body
+//
+// Settings for the `<body>` element.
+$body-bg:                   $white !default;
+$body-color:                $gray-700;
+
+// Links
+//
+// Style anchor elements.
+$link-color:                $primary-darker;
+$link-hover-decoration:     none;
+$link-hover-color:          $primary;
+$link-hover-decoration:     none;
+
+//
+// Typography
+// --------------------------------------------------
+
+$font-family-sans-serif:  "Cachet", sans-serif;
+
+// heading & body font
+$headings-font-family:  $font-family-sans-serif !default;
+$font-family-base:      Verdana, sans-serif;;
```

Comment:
we have this variable, no need to introduce, just use it everywhere, please (addon to comments above)

Reply:
Because we are leveraging Bootstrap 4, we are actually utilizing how it defines the various heading tag font styles. 

In _variables.scss we reassign the variable for $headings-font-family to use the "$font-family-sans-serif" instead of "$font-family-base" which it uses as it's default. 

If we remove the reassigning variable we will need to redefine all h1, .h1, h2, .h2 etc. and add some css to set the font-family in a separate SASS file. It would result in more overriding code. 

This is the cleanest solution IMO.